### ISSUE TITLE
error compiling with @keyframes

### ISSUE THREAD

**Issue by dayjoyx** [laggingreflex/namespace-css-module-loader#1]
my less file was look like: 
```css
@keyframes loadingCircle {
  0% {
    transform-origin: 50% 50%;
    transform: rotate(0deg);
  }
  100% {
    transform-origin: 50% 50%;
    transform: rotate(360deg);
  }
}
```

after webpack:

```js
rules: [{
      resource: {
        include: [SOURCE_PATH, EXAM_PATH],
        test: /\.less$/
      },
      use: [{
        loader: 'style-loader'
      }, {
        loader: 'css-loader'
      }, {
        loader: 'postcss-loader',
        options: {
          plugins: function () {
            return [autoprefixer('> 5%')]
          }
        }
      }, {
        loader: 'namespace-css-module-loader'
      }, {
        loader: 'less-loader'
      }]
    }
]
```

I got:

```css
@keyframes loadingCircle {
  :global .ucDrkdfCPe8N1_G8MhmwR 0% {
    -webkit-transform-origin: 50% 50%;
            transform-origin: 50% 50%;
    -webkit-transform: rotate(0deg);
            transform: rotate(0deg);
  }
  :global .ucDrkdfCPe8N1_G8MhmwR 100% {
    -webkit-transform-origin: 50% 50%;
            transform-origin: 50% 50%;
    -webkit-transform: rotate(360deg);
            transform: rotate(360deg);
  }
}
```

the :global is at wrong place, so all the animations are not working properly. 

Could it possible to fix this? THX

**Comment by laggingreflex** [comment]
Hi there! Sorry for the late reply. 

Seems like a postcss/css-loader issue. https://github.com/webpack-contrib/css-loader/issues/539

**Comment by yoyomo** [comment]
I made a pull request for this  #3

**Comment by laggingreflex** [comment]
Merged #3. Hope that fixes this! Closing.
